src/pps/pps.proto

```diff
@@ -708,7 +709,9 @@ message CreatePipelineRequest {
   Metadata metadata = 28;
   string reprocess_spec = 29;
   bool autoscaling = 30;
+  // TODO: Why is there a field number gap here?
   repeated Toleration tolerations = 34;
+  bool batching = 35;
```

Comment:
So batching is a property of the Pipeline, not the Transform?

Reply:
I expanded on this in the design doc.
> I think datum batching needs to be coupled to the Pipeline Transform as a concept.  We are going to have a situation where some docker images are for batching, and some for non-batching.  This needs to make it into our model.  I believe we call the docker image, and the other parameters that go into running it the Transform, so that seems like the best place.

Reply:
I moved it to the transform.